### PR TITLE
Add unit test for isNoneBlank of stringUtils class

### PR DESCRIPTION
**Pull request by hienkwan**
This PR for adding unit test for isNoneBlank method in stringUtils

Participant : @trgpnt

### CODE REVIEW

File: src/test/java/com/aggregated/StringUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+public class StringUtilsTest {
+
+    @Test
+    public void testIsNonBlankWithBlankString() {
+        assertFalse(StringUtils.isNoneBlank(""));
+    }
+
+    @Test
+    public void testIsNonBlankWittNullString(){
```
Comment (trgpnt) [inline src/test/java/com/aggregated/StringUtilsTest.java:15]: Please consider templating the syntax room,
be it
```
(){
```
or
```
() {
```

Comment (hienkwan) [reply src/test/java/com/aggregated/StringUtilsTest.java]: My bad, I'm forgot to apply code format for it. Done for now.

Comment (trgpnt) [reply src/test/java/com/aggregated/StringUtilsTest.java]: Thanks @hienkwan !

**Review by trgpnt — COMMENTED** [review]
I'm contemplating about establishing a well-defined testing structure for Strings,
1. For test data :
We can use a string randomizer to output a random string with a defined rule (blank, contains [aA-Zz] and [0-9] and [any special char], or contains only the defined characters we input ( only spaces , etc )
2. For code structure:
 I'm considering on this template:
```
@Test
    public void testDummy() {
        int length = 10;
        boolean useLetters = true;
        boolean useNumbers = false;
        String generatedString = RandomStringUtils.random(length, useLetters, useNumbers);
        
        final Function<String, Boolean> testNoneBlank = StringUtils::isNoneBlank;
        assertTrue(testNoneBlank.apply(generatedString));
    }
```

```RandomStringUtils``` is imported from 
[apache-commons-lang](https://mvnrepository.com/artifact/org.apache.commons/commons-lang3/3.12.0) 

We can make use of this, and better serve for other string-related method testing.
What's your suggestion?

**Comment by hienkwan** [comment]
> I'm contemplating about establishing a well-defined testing structure for Strings,
> 
> 1. For test data :
>    We can use a string randomizer to output a random string with a defined rule (blank, contains [aA-Zz] and [0-9] and [any special char], or contains only the defined characters we input ( only spaces , etc ), a partial-buildable string value might be necessary.
> 2. For code structure:
>    I'm considering on this template:
> 
> ```
> @Test
>     public void testDummy() {
>         int length = 10;
>         boolean useLetters = true;
>         boolean useNumbers = false;
>         String generatedString = RandomStringUtils.random(length, useLetters, useNumbers);
>         
>         final Function<String, Boolean> testNoneBlank = StringUtils::isNoneBlank;
>         assertTrue(testNoneBlank.apply(generatedString));
>     }
> ```
> 
> `RandomStringUtils` is imported from [apache-commons-lang](https://mvnrepository.com/artifact/org.apache.commons/commons-lang3/3.12.0)
> 
> I'm planning on BaseTest.java, TestCase.java classes as well.
> 
> sth like : https://examples.javacodegeeks.com/java-development/core-java/junit/junit-setup-teardown-example/ to be more readable and maintainable
> 
> We can make use of this, and better serve for other string-related method testing. What's your suggestion?



> I'm contemplating about establishing a well-defined testing structure for Strings,
> 
> 1. For test data :
>    We can use a string randomizer to output a random string with a defined rule (blank, contains [aA-Zz] and [0-9] and [any special char], or contains only the defined characters we input ( only spaces , etc ), a partial-buildable string value might be necessary.
> 2. For code structure:
>    I'm considering on this template:
> 
> ```
> @Test
>     public void testDummy() {
>         int length = 10;
>         boolean useLetters = true;
>         boolean useNumbers = false;
>         String generatedString = RandomStringUtils.random(length, useLetters, useNumbers);
>         
>         final Function<String, Boolean> testNoneBlank = StringUtils::isNoneBlank;
>         assertTrue(testNoneBlank.apply(generatedString));
>     }
> ```
> 
> `RandomStringUtils` is imported from [apache-commons-lang](https://mvnrepository.com/artifact/org.apache.commons/commons-lang3/3.12.0)
> 
> I'm planning on BaseTest.java, TestCase.java classes as well.
> 
> sth like : https://examples.javacodegeeks.com/java-development/core-java/junit/junit-setup-teardown-example/ to be more readable and maintainable
> 
> We can make use of this, and better serve for other string-related method testing. What's your suggestion?

It's good for me. So let's follow your suggestion

File: src/test/java/com/aggregated/StringUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        assertFalse(StringUtils.isNoneBlank(null));
```
[the remark this answers is not in the document]
Comment (hienkwan) [reply src/test/java/com/aggregated/StringUtilsTest.java:16]: Yup, make sense to me.

File: src/test/java/com/aggregated/StringUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    @Test
+    public void testIsNoneBlankWithLetterOrDigitString() {
+        assertTrue(StringUtils.isNoneBlank("abc123"));
+    }
+
+    @Test
+    public void testIsNoneBlankWithSpecialCharacters() {
+        assertTrue(StringUtils.isNoneBlank("abc!@#"));
```
[the remark this answers is not in the document]
Comment (hienkwan) [reply src/test/java/com/aggregated/StringUtilsTest.java:31]: It's still work fine :)

**Comment by trgpnt** [comment]
@hienkwan , please find and review the [latest commit hash](https://github.com/trgpnt/Java-Class-Annotatable-Constructor-Templater/pull/13/commits/ffe001927bcfd408a71ee9d84899364be865ad9a#diff-836d226bf17e833684da8999f999079ac0ec2bd8082fac9f7fdccbeaa9fff81e) that addressed what we'd previously discussed in this PR.
Thank you for your time.

**Review by trgpnt — APPROVED** [review]
@hienkwan FYI

**Pull request merged**